guides/v2.3/graphql/tutorials/checkout/checkout-billing-address.md

```diff
@@ -0,0 +1,316 @@
+            save_in_address_book: false
+          }
+          use_for_shipping: true
+      }
+    }
+  ) {
+    cart {
+      billing_address {
+        firstname
+        lastname
+        company
+        street
+        city
+        postcode
+        telephone
+        country {
+          code
+          label
+        }
+        address_type
+      }
+      shipping_addresses {
+        firstname
+        lastname
+        company
+        street
+        city
+        postcode
+        telephone
+        country {
+          code
+          label
+        }
+        address_type
+      }
+    }
+  }
+}
+```
+
+**Response**
+
+```json
+{
+  "data": {
+    "setBillingAddressOnCart": {
+      "cart": {
+        "billing_address": {
+          "firstname": "John",
+          "lastname": "Doe",
+          "company": "Company Name",
+          "street": [
+            "320 N Crescent Dr",
+            "Beverly Hills"
+          ],
+          "city": "Los Angeles",
+          "postcode": "90210",
+          "telephone": "123-456-0000",
+          "country": {
+            "code": "US",
+            "label": "US"
+          },
+          "address_type": "BILLING"
+        },
+        "shipping_addresses": [
+          {
+            "firstname": "John",
+            "lastname": "Doe",
+            "company": "Company Name",
+            "street": [
+              "320 N Crescent Dr",
+              "Beverly Hills"
+            ],
+            "city": "Los Angeles",
+            "postcode": "90210",
+            "telephone": "123-456-0000",
+            "country": {
+              "code": "US",
+              "label": "US"
+            },
+            "address_type": "SHIPPING"
+          }
+        ]
+      }
+    }
+  }
+}
+```
+
+### Use an existing customer address
+
+First, query the customer to return the list of address IDs.
+
+**Request**
+
+```text
+query {
+  customer {
+    addresses {
+      id
+      default_billing
+      default_shipping
+    }
+  }
+}
+```
+
+**Response**
+
+```text
+  "data": {
+    "customer": {
+      "addresses": [
+        {
+          "id": 2,
+          "default_billing": true,
+          "default_shipping": true
+        }
+      ]
+    }
+  }
+}
+```
+
+Set {{ CUSTOMER_ADDRESS_ID }}  to an `id` returned in the query.
+
+{{ CART_ID }}` is the unique shopping cart ID from [Step 2. Create empty cart]({{ page.baseurl }}/graphql/tutorials/checkout/checkout-add-product-to-cart.html).
+
+**Request**
+
+```text
+mutation {
+  setBillingAddressOnCart(
+    input: {
+      cart_id: "{{ CART_ID }}"
+      billing_address: {
+          customer_address_id: {{ CUSTOMER_ADDRESS_ID }}
+      }
+    }
+  ) {
+    cart {
+      billing_address {
+        firstname
+        lastname
+        company
+        street
+        city
+        postcode
+        telephone
+        country {
+          code
+          label
+        }
+        address_type
+      }
+    }
+  }
+}
+```
+
+where
```

Comment:
Delete these lines. The variables are now discussed before the example.

Reply:
**fixed**